hr_employee_exemption/models/hr_employee.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding:utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Savoir-faire Linux. All Rights Reserved.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import api, models, fields
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    exemption_ids = fields.One2many(
+        'hr.employee.exemption',
+        'employee_id',
+        'Income Tax Exemptions',
+        groups='base.group_hr_manager',
+    )
+
```

Comment:
If someone is exempted from a tax, it has nothing to do with his contract. The person would be exempted whatever job he has.